src/components/RichTextEditor/index.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+import { Editor, EditorState, RichUtils } from 'draft-js';
+import BlockStyleControls from '../BlockStyleControls';
+import InlineStyleControls from '../InlineStyleControls';
+
+export default class RichTextEditor extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { editorState: EditorState.createEmpty() };
```

Comment:
No components should have state, but because I don't know how this specific components works we can discuss it. To me this seems that value of this rich component is inside this and not in Redux store which prevents us from taking this value when we want to take it because it's trapped inside this components state which is against Redux principle.